src/libsync/account.h

```diff
@@ -275,6 +279,15 @@ class OWNCLOUDSYNC_EXPORT Account : public QObject
 
     std::shared_ptr<UserStatusConnector> userStatusConnector() const;
 
+    void setLockFileState(const QString &serverRelativePath,
+                          SyncJournalDb *journal,
+                          const SyncFileItem::LockStatus lock);
```

Comment:
@mgallien 
`const SyncFileItem::LockStatus lock` -> `const SyncFileItem::LockStatus lockStatus`/`const SyncFileItem::LockStatus lockState`. Just for sake of readability.